packages/contracts/script/DeploymentConfig.s.sol

```diff
@@ -40,11 +40,15 @@ contract DeploymentConfig is Script {
     address internal SNT_ADDRESS_OPTIMISM_MAINNET = 0x650AF3C15AF43dcB218406d30784416D64Cfb6B2;
     // solhint-disable-next-line var-name-mixedcase
     address internal SNT_ADDRESS_OPTIMISM_GOERLI = 0xcAD273fA2bb77875333439FDf4417D995159c3E1;
+    // solhint-disable-next-line var-name-mixedcase
+    address internal SNT_ADDRESS_OPTIMISM_SEPOLIA = 0x0B5DAd18B8791ddb24252B433ec4f21f9e6e5Ed0;
 
     // solhint-disable-next-line var-name-mixedcase
     address internal MULTICALL_ADDRESS_OPTIMISM = 0xeAa6877139d436Dc6d1f75F3aF15B74662617B2C;
     // solhint-disable-next-line var-name-mixedcase
     address internal MULTICALL_ADDRESS_OPTIMISM_GOERLI = 0xcA11bde05977b3631167028862bE2a173976CA11;
+    // solhint-disable-next-line var-name-mixedcase
+    address internal MULTICALL_ADDRESS_OPTIMISM_SEPOLIA = 0xcA11bde05977b3631167028862bE2a173976CA11;
```

Comment:
non-blocking: do you know why `MULTICALL_ADDRESS_OPTIMISM_SEPOLIA` and `MULTICALL_ADDRESS_OPTIMISM_GOERLI` have the same address?